### PR TITLE
Updates for content bundle 

### PR DESCRIPTION
**Pull request by Rania-Amr**
buganizer ticket: b/356075164
### Removing

- key: folder_name: https://github.com/search?q=org%3ACloudVLab++key%3A+folder_name+-repo%3ACloudVLab%2Fqwiklabs-test-content+-repo%3ACloudVLab%2Fqwiklabs-content-bundle-spec+&type=code
### Adding 
- Adding Max_duration: https://github.com/CloudVLab/gcp-spl-content/blob/e0bc019025d0007cad490e99a8d8f5475a3c7e8b/labs/gsp000-ghost/qwiklabs.yaml#L6

- Adding Credits: https://github.com/CloudVLab/gcp-spl-content/blob/e0bc019025d0007cad490e99a8d8f5475a3c7e8b/labs/mini-lab-48/qwiklabs.yaml#L10

- Allowed_locations_constraint https://github.com/CloudVLab/gcp-ce-content/blob/ca8feb25013c16520f8c76b816e8b4d13b54cf04/labs/ceCBL472-VPCandGCE/qwiklabs.yaml#L34

- variant: gcp_only: https://github.com/CloudVLab/gcp-spl-content/blob/e727ab73c64d78334f8a1ad0ae8ebf01d1f441b3/labs/gsp1211-a-tour-networking/qwiklabs.yaml#L21

- cloud_terminal: https://github.com/CloudVLab/gcp-spl-content/blob/e727ab73c64d78334f8a1ad0ae8ebf01d1f441b3/labs/mini-lab-48/qwiklabs.yaml#L18]

- linux_terminal:https://github.com/CloudVLab/gcp-it-cert-content/blob/62ca90265f23f66dbfcf64b2331e00039d9e997d/labs/gic221bb-process-text-files-with-python-dicts-and-upload-to-running-web-service/qwiklabs.yaml#L24

- jupyter_notebook: https://github.com/CloudVLab/gcp-it-cert-content/blob/62ca90265f23f66dbfcf64b2331e00039d9e997d/labs/ada007-explore_raw_data/qwiklabs.yaml#L19

- windows_vm:https://github.com/CloudVLab/gcp-enablement-content/blob/e57065f76f16158f297803966cfba33529903a59/labs/cepf018-infra-mod-stratozone-assessment/qwiklabs.yaml#L71

- looker_instance: https://github.com/CloudVLab/gcp-spl-content/blob/e727ab73c64d78334f8a1ad0ae8ebf01d1f441b3/labs/sample-looker-lab/qwiklabs.yaml#L10
-

Testing lab 
- https://davesr.dev.qwiklabs.com/authoring/labs/1295950
  - https://github.com/CloudVLab/qwiklabs-test-content/blob/6c514845ea64d263dd4595e4574282b07bf8e71e/labs/test-content-bundle-lab/qwiklabs.yaml
-  https://davesr.dev.qwiklabs.com/labs/388569/edit
   - https://github.com/CloudVLab/gcp-spl-content/blob/d1fd5614203e7b64985b5da7fe8eae057f27ce97/labs/sample-looker-lab/qwiklabs.yaml#L10

### CODE REVIEW

File: examples/lab-robust/v2/qwiklabs.yaml [hunk 1]
```diff
@@ -5,11 +5,22 @@ title: Robust Lab Example
...
 tags:
 - sample
 - life-changing
 - gcp
+product_tags: ['app engine', 'kubernetes engine', 'cloud functions']
+role_tags: ['cloud architect', 'developers backend', 'developers frontend']
+domain_tags:
+- infrastructure
+- application development
+- data
```
Comment (tkurpios) [inline examples/lab-robust/v2/qwiklabs.yaml:21]: Are product_tags, role_tags, domain_tags ever used? I remember failing to find an example. Maybe @robsideshow knows. If not, we should not add them to the example.

File: examples/lab-robust/v2/qwiklabs.yaml [hunk 2]
```diff
@@ -105,12 +173,21 @@ environment:
   - label: GCP Folder Display Name
     reference: primary_folder.display_name
 
+instruction:
+  type: html
```
Comment (tkurpios) [inline examples/lab-robust/v2/qwiklabs.yaml:177]: That's being developed by @julianaviola atm.

File: examples/lab-robust/v2/qwiklabs.yaml [hunk 3]
```diff
@@ -5,11 +5,22 @@ title: Robust Lab Example
...
 - sample
 - life-changing
 - gcp
+product_tags: ['app engine', 'kubernetes engine', 'cloud functions']
+role_tags: ['cloud architect', 'developers backend', 'developers frontend']
+domain_tags:
+- infrastructure
+- application development
+- data
+legacy_display_options:
```
Comment (tkurpios) [inline examples/lab-robust/v2/qwiklabs.yaml:22]: Just double-checking without understanding how we handle it - this sounds like a property that we put to old labs for some legacy reasons, I'm not sure we would want to include this in the new labs and hence not sure if that's sth we want to promote here. Have you found the meaning of this property?

File: examples/lab-robust/v2/qwiklabs.yaml (hunk 1 above)
Comment (Rania-Amr) [reply examples/lab-robust/v2/qwiklabs.yaml:21]: - Recently product_tags, role_tags, domain_tags are not used. you can check that [here](https://github.com/CloudVLab/gcp-brex/blob/b22611ca59b5ede434a79b50d419158b4901e5be/labs/qt_oct23_week3/qwiklabs.es.yaml#L10).
This is cl for adding this attribute to [g3](https://critique.corp.google.com/cl/324780113/depot/google3/cloud/training/qwiklabs/content_bundle/examples/lab-v1-robust/qwiklabs.yaml)
- For legacy_display_options last time we used this attribute from [4 years ](https://github.com/CloudVLab/qwiklabs-test-content/blob/ff5271ea63f5fa1a8b943d4c87e8ab32690ad861/labs/test_v1_lab_migration_5/qwiklabs.yaml#L10)
this [Cl](https://critique.corp.google.com/cl/258640416/depot/google3/cloud/training/qwiklabs/content_bundle/schema/content_bundle_lab.yaml) for that

File: examples/lab-robust/v2/qwiklabs.yaml (hunk 3 above)
Comment (Rania-Amr) [reply examples/lab-robust/v2/qwiklabs.yaml:22]: - For legacy_display_options last time we used this attribute from [4 years ](https://github.com/CloudVLab/qwiklabs-test-content/blob/ff5271ea63f5fa1a8b943d4c87e8ab32690ad861/labs/test_v1_lab_migration_5/qwiklabs.yaml#L10)
this [Cl](https://critique.corp.google.com/cl/258640416/depot/google3/cloud/training/qwiklabs/content_bundle/schema/content_bundle_lab.yaml) for that
Should i remove it?

File: examples/lab-robust/v2/qwiklabs.yaml (hunk 1 above)
Comment (tkurpios) [reply examples/lab-robust/v2/qwiklabs.yaml:21]: If they're not used, I think we should not add them to the example

**Review by tkurpios — CHANGES_REQUESTED** [review]
Have you managed to verify that this lab works correctly and respects the changes? If so, feel free to merge

File: examples/lab-robust/v2/qwiklabs.yaml (hunk 3 above)
Comment (tkurpios) [reply examples/lab-robust/v2/qwiklabs.yaml]: The only usages are in test repos and in archived labs: 
https://github.com/search?q=org%3ACloudVLab+legacy_display_options+-repo%3ACloudVLab%2Fqwiklabs-test-content+-repo%3ACloudVLab%2Fqwiklabs-content-bundle-spec+&type=code

Please add a buganizer issue to remove this property entirely.

File: examples/lab-robust/v2/qwiklabs.yaml (hunk 2 above)
Comment (robsideshow) [reply examples/lab-robust/v2/qwiklabs.yaml:169]: +1 i.e. I would not include `html` right now.

**Comment by Rania-Amr** [comment]
@tkurpios @robsideshow I updated the pull request with examples of existing labs for new keys

**Comment by tkurpios** [comment]
> Have you managed to verify that this lab works correctly and respects the changes? If so, feel free to merge

I didn't get response to this one. Probably the easiest way to do that is to copy it to `qwiklabs-test-content` repo and see if it was created correctly in `davesr`

**Comment by Rania-Amr** [comment]
> > Have you managed to verify that this lab works correctly and respects the changes? If so, feel free to merge
> 
> I didn't get response to this one. Probably the easiest way to do that is to copy it to `qwiklabs-test-content` repo and see if it was created correctly in `davesr`
I didn't create a new lab with all changes but i provided examples with existing labs they are using this attribute

**Comment by tkurpios** [comment]
> > > Have you managed to verify that this lab works correctly and respects the changes? If so, feel free to merge
> > 
> > 
> > I didn't get response to this one. Probably the easiest way to do that is to copy it to `qwiklabs-test-content` repo and see if it was created correctly in `davesr`
> > I didn't create a new lab with all changes but i provided examples with existing labs they are using this attribute

I'm concerned that there might be some typo or other issue that prevents this lab from being created. Would you mind testing it in `qwiklabs-test-content` repo before merging this PR? That should be straightforward hopefully.

**Review by tkurpios — APPROVED** [review]

**Pull request merged**